### PR TITLE
Add parentheses to arrow functions single parameters (part 3)

### PR DESCRIPTION
**Pull request by teoli2003**
See #18677

### CODE REVIEW

File: files/en-us/web/api/imagedecoder/index.md [hunk 1]
```diff
@@ -79,12 +79,12 @@ function renderImage(result) {
   // Decode the next frame ahead of display so it's ready in time.
   imageDecoder.decode({frameIndex: ++imageIndex})
       .then(
-          nextResult => setTimeout(
-              _ => {
+          (nextResult) => setTimeout(
+              (_) => {
```
Comment (Josh-Cena) [inline files/en-us/web/api/imagedecoder/index.md:83]:
```suggestion
              () => {
```

File: files/en-us/web/api/media_session_api/index.md [hunk 2]
```diff
@@ -87,8 +87,8 @@ playButton.addEventListener('pointerup', function(event) {
 
   // User interacted with the page. Let's play audio!
   audio.play()
-  .then(_ => { /* Set up media session controls, as shown above. */ })
-  .catch(error => { console.log(error) });
+  .then((_) => { /* Set up media session controls, as shown above. */ })
```
Comment (Josh-Cena) [inline files/en-us/web/api/media_session_api/index.md:90]:
```suggestion
  .then(() => { /* Set up media session controls, as shown above. */ })
```

**Comment by teoli2003** [comment]
I kept the `(_)`, but you are right, `()` is better. I will fix them from now on and do a quick check at the end.

**Review by Josh-Cena — APPROVED** [review]
Apart from that, LGTM! You can merge if you think it's ready.

**Pull request merged**